rtb/norme-di-progetto/src/gestione-qualita.tex

```diff
@@ -1,7 +1,43 @@
+\pagebreak % aggiunto per riuscire a caricare la pagina
 \subsection{Gestione della Qualità}
 
 \subsubsection{Descrizione, scopo ed aspettative}
+La gestione della qualità di progetto è l'insieme dei processi e delle attività che vengono eseguite per garantire che la qualità del prodotto finale 
```

Comment:
di progetto ?

Reply:
dici togliere direttamente "di progetto"? Forse suona meglio in effetti